webapp/pgconfig.py

```diff
@@ -1,466 +1,462 @@
+    into the `short_desc` column.
+
+    Note: This is useful for added and removed items, NOT changed items.
 
     Parameters
-    ---------------
-    df : pandas.DataFrame
+    ---------------------
+    data : pd.DataFrame
+    original_columns : list
+    new_column : str
 
     Returns
-    ---------------
-    str
-        HTML table for display.
+    ---------------------
+    data_new : pd.DataFrame
     """
-    classes = ['table', 'table-hover']
-    html_raw = '<div id="config_table">{src}</div>'
-    src = df.to_html(index=False,
-                     classes=classes,
-                     justify='center',
-                     escape=False)
-    html = html_raw.format(src=src)
-    return html
+    data['tmp'] = data[original_columns].bfill(axis=1).iloc[:, 0]
+    data_mid = data.drop(columns=original_columns)
+    data_new = data_mid.rename(columns={'tmp': new_column})
+    return data_new
 
-def config_changes(config_summary, vers1, vers2):
-    """Uses `config_summary` to find changes between `vers1` and `vers2`.
 
-    Parameters
-    ----------------
-    config_summary : pandas.DataFrame
-        Full data set with all loaded versions.
+def load_config_data(pg_version: int) -> pd.DataFrame:
+    """Loads the pickled config data for `pg_version` into DataFrame with the
+    config name as the index.
 
-    vers1 : str
-        Version number in str format.  e.g. 11 or 9_6
+    Returns empty DataFrame on file read error.
 
-    vers2 : str
-        Version number in str format.  e.g. 11 or 9_6
+    Parameters
+    ----------------------
+    pg_version : int
 
     Returns
-    ----------------
-    changes : pandas.DataFrame
-        Subset of data from `config_summary` for `vers1` and `vers2`.
-    """
-    try:
-        data = config_summary[['parameter', vers1, vers2]]
-    except KeyError as err:
-        LOGGER.error('Config version not found. %s', err)
-        raise
-    changes = data[(data[vers1] != data[vers2])
-                   & (data[vers1].notnull() | data[vers2].notnull())]
-    return changes
+    ----------------------
+    df : pd.DataFrame
+    """ 
+    base_path = os.path.dirname(os.path.realpath(__file__))
+    # Checking user input against configured versions to avoid security concerns
+    if str(pg_version) not in VERSIONS:
+        raise ValueError(f'Invalid Postgres version.  Options are {VERSIONS}')
 
-def config_changes_html(changes):
-    """Converts `changes` DataFrame into HTML table.
-
-    Additionally, creates column with URL to parameter's history over versions
-    of Postgres.
-
-    Parameters
-    ---------------
-    changes : pandas.DataFrame
+    filename = os.path.join(base_path, 'config', f'pg{pg_version}.pkl')
 
-    Returns
-    ---------------
-    html : str
-        HTML table for display.
-    """
-    # Add `history` column with URL to param history (if exists)
+    try:
+        with open(filename, 'rb') as data_file:
```

Comment:
I already dismissed this alert..... :roll_eyes: 

The user input is validated against a set of valid options and errors out if invalid input.  

```
    # Checking user input against configured versions to avoid security concerns
    if str(pg_version) not in VERSIONS:
        raise ValueError(f'Invalid Postgres version.  Options are {VERSIONS}')
```